modules/tests/example1.scala

```diff
@@ -0,0 +1,69 @@
+/* -
```

Comment:
I think all examples should go in tut files as parts of the docs or they'll be invisible for most casual users